lib/howitzer/meta/iframe.rb

```diff
@@ -0,0 +1,35 @@
+module Howitzer
+  module Meta
+    # This class represents iframe entity within howitzer meta information interface
+    class Iframe < Base
```

Comment:
I have big doubts about this class implementation. I see extra complexity related with frames. What about removing it from first version